src/data/atlas-search-index.ts

```diff
@@ -98,7 +126,7 @@ export const SearchIndex: IndexMappings = {
       source: {
         collection: SYNONYM_COLLECTION_NAME,
       },
-      analyzer: 'lucene.keyword',
+      analyzer: 'lucene.whitespace',
```

Comment:
If we think it's worthwhile, we could make a custom analyzer to address the issue of capitalization with the whitespace analyzer.

Here's what it could look like:

```json
"analyzers": [
  {
    "name": "synonymAnalyzer",
    "tokenizer": {
      "type": "whitespace"
    },
    "tokenFilters": [
         {
             "type": "lowercase" 
         }
     ]
  }
]
```

You should be able to reference this custom analyzer here with the `name` property.

[custom analyzer reference](https://www.mongodb.com/docs/atlas/atlas-search/analyzers/custom/)
[lowercase token filter reference](https://www.mongodb.com/docs/atlas/atlas-search/analyzers/token-filters/#lowercase)

Reply:
great call @branberry ! we can also expand on this analyzer in the future (ie. when adding multi whitespace words)

+1 on this suggestion

Reply:
Great idea, thanks @branberry!! I updated it and it fixed the capitalization issue. 

Also @seungpark, multi whitespace word synonyms are already working so that ~shouldn't be necessary :)